### PR TITLE
[feature] Create an internal Docs Guide for repositories

### LINKED ISSUE #299: [feat] Internal Docs Style Guide
**Is your feature request related to a problem? Please describe.**
I noticed that there is no specific style guide to maintain the Documentation for the repos. 

**Describe the solution you'd like**
Create an internal docs style guide. It would not only help people gain experience in working with docs, but also makes the content appear organized. 

**Describe alternatives you've considered**
Nothing really. 

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by CBID2**
## Link to issue
Closes #299

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [X] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [X] I ran the app and tested it locally to verify that it works as expected.
- [X] I have checked my code with an automatic accessibility tool such as Axe Dev Tools or Wave 
      and it shows no errors.
## Additional Information (Optional)
Any additional information that you want to give us.

### CODE REVIEW

File: pages/docs/Style Guide.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Using the active voice in writing makes your message more direct and engaging.
+
+- ❌ **Not this:** Gitpod **was** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+- ✅ **Use this:** Gitpod **is** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
+
+### Jargon
+
+Using technical jargon can alienate non-technical readers.
+
+- ❌ **Not this:** Gitpod **was** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
```
Comment (Neha) [inline pages/docs/Style Guide.md:61]: seems like this example here is not the example of Jargon

File: pages/docs/Style Guide.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+### Headings
+
+- Use descriptive titles for headings.
+  - ❌ **Not this:** `<h1> Yoga</h1>`
+- ✅ **Use this:** `<h1>Yoga for Developers</h1>`
+
+- Place headings in chronological order.
+- ❌ **Not this:** `<h1>Yoga for Developers</h1>`
+  `<h2>What is Yoga?</h2>`
```
Comment (Neha) [inline pages/docs/Style Guide.md:96]: I think two `h2` can follow each other but it should not be: 

```HTML
<h1>Heading one</h1>
<h3>Heading one</h3>
<h2>Heading 2</h2>
```

**Review by Neha — COMMENTED** [review]
Hey, I left 2 small comments (observations) . Thanks

File: pages/docs/Style Guide.md (hunk 1 above)
Comment (CBID2) [reply pages/docs/Style Guide.md:61]: Good point @Neha! Do you have an other suggestions?

File: pages/docs/Style Guide.md (hunk 2 above)
Comment (CBID2) [reply pages/docs/Style Guide.md:96]: Noted

File: pages/docs/Style Guide.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  - ❌ **Not this:** Join the conversation on our Discord community! 😀
+  - ✅ **Use this:** Join the conversation on our Discord community!
+
+### Headings
+
+- Use descriptive titles for headings.
+  - ❌ **Not this:** `<h1> Yoga</h1>`
+- ✅ **Use this:** `<h1>Yoga for Developers</h1>`
+
+- Place headings in chronological order.
```
Comment (EmmaDawsonDev) [inline pages/docs/Style Guide.md:94]:
```suggestion
- Place headings in sequential order.
```

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
This is fantastic Chrissy! I've just left a few small comments. Also, the structure of this repo isn't the same as linkFree so there's no documentation on the website yet. (Maybe it's something we could add at a later date). Would you be able to move this file to the root of the project?

**Comment by CBID2** [comment]
> This is fantastic Chrissy! I've just left a few small comments. Also, the structure of this repo isn't the same as linkFree so there's no documentation on the website yet. (Maybe it's something we could add at a later date). Would you be able to move this file to the root of the project?

Hey @EmmaDawsonDev! Thanks for the suggestion. When you say root, what does that mean?

File: pages/docs/Style Guide.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+## Clarity 🤔
+
+Clarity in writing helps ensure that your audience understands your message and intentions. Here are some best practices to follow:
+
+### Active vs. Passive
+
+Using the active voice in writing makes your message more direct and engaging.
+
+- ❌ **Not this:** Gitpod **was** an open-source application that provides a prebuilt development environment in your browser - powered by VS Code.
```
[the remark this answers is not in the document]
Comment (CBID2) [reply pages/docs/Style Guide.md:54]: Good idea!

**Comment by EmmaDawsonDev** [comment]
> > This is fantastic Chrissy! I've just left a few small comments. Also, the structure of this repo isn't the same as linkFree so there's no documentation on the website yet. (Maybe it's something we could add at a later date). Would you be able to move this file to the root of the project?
> 
> Hey @EmmaDawsonDev! Thanks for the suggestion. When you say root, what does that mean?

Sorry for not being clear! I mean in the top-level folder of the whole repository. It should be at the same level as things like the Code of Conduct and the readme.

**Comment by CBID2** [comment]
> > > This is fantastic Chrissy! I've just left a few small comments. Also, the structure of this repo isn't the same as linkFree so there's no documentation on the website yet. (Maybe it's something we could add at a later date). Would you be able to move this file to the root of the project?
> 
> > 
> 
> > Hey @EmmaDawsonDev! Thanks for the suggestion. When you say root, what does that mean?
> 
> 
> 
> Sorry for not being clear! I mean in the top-level folder of the whole repository. It should be at the same level as things like the Code of Conduct and the readme.

Ahh [so in here](https://github.com/AccessibleForAll/AccessibleWebDev/tree/main) basically?

**Comment by EmmaDawsonDev** [comment]
Yes :)

**Comment by CBID2** [comment]
> Yes :)

Ok thanks

**Review by EmmaDawsonDev — APPROVED** [review]
This is fab! Thanks a lot Chrissy!

**Comment by CBID2** [comment]
> This is fab! Thanks a lot Chrissy!

Anytime @EmmaDawsonDev! :)

**Pull request merged**